### PR TITLE
increase lockup button

### PR DESCRIPTION
**Pull request by msavir**
Owner can increase lockup through the UI

image

### CODE REVIEW

**Comment by zihan-aptos** [comment]
Would an owner want to increase lockup before staking? It might be better to make increase lockup a separate page so that owners can do it before staking too. Also increasing lockup is a different flow/feature/function than staking and it makes more sense to have it own page.

**Comment by msavir** [comment]
> Would an owner want to increase lockup before staking? It might be better to make increase lockup a separate page so that owners can do it before staking too. Also increasing lockup is a different flow/feature/function than staking and it makes more sense to have it own page.

this page is **after** initialize staking. Only accounts that have staking pool see this page. It makes sense to have this page as the "owner dashboard" where they can make changes to their staking pool imo

**Comment by msavir** [comment]
> Would an owner want to increase lockup before staking? It might be better to make increase lockup a separate page so that owners can do it before staking too. Also increasing lockup is a different flow/feature/function than staking and it makes more sense to have it own page.

I am not sure they can increase lockup without having a StakePool resource....?

**Comment by zihan-aptos** [comment]
> > Would an owner want to increase lockup before staking? It might be better to make increase lockup a separate page so that owners can do it before staking too. Also increasing lockup is a different flow/feature/function than staking and it makes more sense to have it own page.
> 
> I am not sure they can increase lockup without having a StakePool resource....? Having the increase lockup on this page makes sense as this page is kind of an "owner dashboard" where they can make changes to their staking pool imo

@movekevin please let us know if owners can/need increase lockup

File: src/pages/Governance/Stake/Edit.tsx [hunk 1]
```diff
@@ -12,34 +13,41 @@ type EditProps = {
...
+    <>
+      <Grid container spacing={6}>
+        <Grid item xs={12} sm={12}>
+          <EditOperator
+            operatorAddress={operator_address}
+            isWalletConnected={isWalletConnected}
+          />
+        </Grid>
+        <Grid item xs={12} sm={12}>
+          <hr />
```
Comment (zihan-aptos) [inline src/pages/Governance/Stake/Edit.tsx:33]: Can we use `<Divider variant="dotted" orientation="horizontal" />`

File: src/pages/Governance/Stake/components/IncreaseLockup.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      <TransactionResponseSnackbar
+        transactionResponse={transactionResponse}
+        onCloseSnackbar={onCloseSnackbar}
+      />
+      <LoadingModal open={transactionInProcess} />
+      <Grid>
+        <Grid container spacing={4}>
+          <></>
+          <Grid item xs={12} sm={8}>
+            Locked until: {renderTimestamp(lockedUntilSec)}
```
Comment (zihan-aptos) [inline src/pages/Governance/Stake/components/IncreaseLockup.tsx:52]:
```suggestion
            LOCKED UNTIL: {renderTimestamp(lockedUntilSec)}
```

**Review by zihan-aptos — APPROVED** [review]

File: src/pages/Governance/Stake/components/IncreaseLockup.tsx [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  );
+
+  return (
+    <>
+      <TransactionResponseSnackbar
+        transactionResponse={transactionResponse}
+        onCloseSnackbar={onCloseSnackbar}
+      />
+      <LoadingModal open={transactionInProcess} />
+      <Grid>
```
Comment (zihan-aptos) [inline src/pages/Governance/Stake/components/IncreaseLockup.tsx:48]: can we set alignItems to `center` for the text and the button?

**Review by movekevin — APPROVED** [review]

**Pull request merged**